juniper_official/routing/check-te-rsvp-global-errors.rule

```diff
@@ -0,0 +1,481 @@
+                        status {
+                            color red;
+                            message "Message out of order error count($message-out-of-order)increasing";
+                        }
+                        next;
+                    }
+                }
+                term out-path-error-messages {
+                    when {
+                        increasing-at-least-by-value "$out-path-error-messages" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Out path error messages count($out-path-error-messages)increasing";
+                        }
+                        next;
+                    }
+                }
+                term out-reservation-error-messages {
+                    when {
+                        increasing-at-least-by-value "$out-reservation-error-messages" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Out reservation error messages count($out-reservation-error-messages)increasing";
+                        }
+                        next;
+                    }
+                }
+                term is-received-nack {
+                    when {
+                        increasing-at-least-by-value "$received-nack" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Received nack error count($received-nack)increasing";
+                        }
+                        next;
+                    }
+                }
+                term recv-pkt-disabled-intf {
+                    when {
+                        increasing-at-least-by-value "$recv-pkt-disabled-intf" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Recv-pkt-disabled-intf error count($recv-pkt-disabled-intf)increasing";
+                        }
+                        next;
+                    }
+                }
+                term send-failure {
+                    when {
+                        increasing-at-least-by-value "$send-failure" {
+                            value 1;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Send failure error count($send-failure)increasing";
+                        }
+                        next;
```

Comment:
Since all the states are related to red...why do we want to do next here?

Reply:
Next is needed so all the terms are processed, else we will have to add seperate triggers